workspaces/adr/plugins/adr-common/package.json

```diff
@@ -43,7 +43,8 @@
     "@backstage/integration": "^1.13.0",
     "@backstage/plugin-search-common": "^1.2.13",
     "front-matter": "^4.0.2",
-    "luxon": "^3.0.0"
+    "luxon": "^3.0.0",
+    "marked": "^12.0.0"
```

Comment:
![Optional: fyi](https://gh-labels.dancrews.com/optional/fyi.svg)
This is currently used [here](https://github.com/backstage/community-plugins/blob/90567e287ba1eb2d0353689b76a61624282b98ca/workspaces/adr/plugins/adr-common/src/search.ts#L8), but not installed.